### PR TITLE
prettier formatting, add card validation, date format, and more

### PR DESCRIPTION
**Pull request by jacobragsdale**

### CODE REVIEW

File: src/App.tsx [hunk 1]
```diff
@@ -1,33 +1,37 @@
...
         <Provider store={store}>
-            <Router>
-                <CssBaseline />
-                <Route exact path="/" component={Lander} />
-                <Route exact path="/signin" component={SignIn} />
-                <Route exact path="/project" component={Project} />
-                <Route exact path="/dashboard" component={Dashboard} />
-                <Route exact path="/help" component={Help} />
-            </Router>
+            <MuiPickersUtilsProvider utils={MomentUtils}>
```
Comment (elibullockpapa) [inline src/App.tsx:25]: yaaaaay this if for the date picker right?

Comment (jacobragsdale) [reply src/App.tsx:25]: yes, date picker is implemented in add card modal

File: src/components/DDUtils.ts [hunk 2]
```diff
@@ -1,39 +1,34 @@
...
         if (user && user.displayName) {
-            return user.displayName.split(' ')[0]
+            return user.displayName.split(' ')[0];
         } else {
-            return 'Sign In'
+            return 'Sign In';
         }
     }
-
-    static validateEmail(email: string): boolean {
```
Comment (elibullockpapa) [inline src/components/DDUtils.ts:34]: why did you delete this?

Comment (jacobragsdale) [reply src/components/DDUtils.ts:34]: we're not using it anywhere. Material UI components have built in email validation.

**Review by elibullockpapa — APPROVED** [review]
so much prettier
